pkg/apis/core/types.go

```diff
@@ -2511,6 +2511,11 @@ type PodSecurityContext struct {
 	// +k8s:conversion-gen=false
 	// +optional
 	HostIPC bool
+	// Use the host's user namespace.
+	// Optional: Defaults to empty.
+	// +k8s:conversion-gen=false
+	// +optional
+	HostUserNamespace *bool
```

Comment:
* drop this field in pkg/api/pod#DropDisabledAlphaFields if the feature gate isn't enabled
* ensure this is nil in pkg/apis/core/validation#ValidatePodSpec if the feature gate isn't enabled